### DISCUSSION TITLE
Default PDF viewer

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#33] [Ideas] [answered]
The following secure PDF viewer is being developed by GrapheneOS. DivestOS should include this app in the system by default.

https://github.com/GrapheneOS/PdfViewer

**Comment by MarkCyberSecDev93** [comment]
Yes, I agree. They should also add some of the feature that GrapheneOS or CalyxOS had.

**Reply by rsssxd** [reply]
DivestOS has many features taken from these projects. Details at the link below
https://divestos.org/index.php?page=technical_details

**Reply by MarkCyberSecDev93** [reply]
> DivestOS has many features taken from these projects. Details at the link below https://divestos.org/index.php?page=technical_details

Yes, I read them earlier. There are some features that needs to be added to  improve the security and privacy (a bit) of the ROM (DivestOS). I already posted it earlier about the security improvement feature that GrapheneOS has.

**Comment by SkewedZeppelin** [comment, marked as answer]
I see no reason to include a PDF viewer by default.

**Reply by rsssxd** [reply]
You can enable PDF content to be viewed in a more secure environment.

**Reply by SkewedZeppelin** [reply]
It would be better to see that app available on F-Droid for everyone to use.
https://gitlab.com/fdroid/rfp/-/issues/1163

**Reply by rsssxd** [reply]
The source code is old, but the application received an update in 2021.
https://f-droid.org/en/packages/org.ninthfloor.copperpdf/
